### DISCUSSION TITLE
資安需要做好自己的帳號管理!

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#576] [General]
資訊安全，絕不只是下載防火牆即可。有時沒有做好管理的帳號也會被駭客趁虛而入。
近期企業身分安全治理解決方案公司 SailPoint說明，很多企業的員工數往往不多，但是他們的公司帳號一人可能不只一個。當他們離職時，若這些帳好沒有完整清除，就會給駭客一個入口。文章中就指出曾有一名駭客非常幸運地駭到一位權限相當高的「孤兒帳號」中，也因此他利用此帳號取得非常多該企業內部的資訊，可想而知企業資訊都將散布出去，實為可怕。
若有用不到的帳號，要定時清理內容，如果沒有再用到，也要徹底的刪除掉! 

文章來源:https://technews.tw/2022/04/26/sailpoint/

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，未來會好好處理帳號問題。

**Comment by liangyinglly** [comment]
身份地位越高，越要小心自己帳號管理的安全

**Comment by PoooYen** [comment]
資安依然是國安問題

**Comment by gnsJhenJie** [comment]
企業如果可以導入單一簽入(SSO)的機制集中管理帳號權限的話，就可以避免一個人擁有多個帳號的狀況發生。

**Comment by cordydeng** [comment]
原來做好資安不只是有防火牆就好了，看來email帳號還是不要申請太多支比較好XD

**Comment by HsuChen88** [comment]
各個帳號內管理的密碼是不是也需要注意被盜取

**Comment by handsomeniko** [comment]
推推實用 謝謝分享

**Comment by Sazmafuji** [comment]
網路安全真要小心

**Comment by Rosancl** [comment]
推，除了平常做好密碼管理、防火牆等，往往會忽略掉不常用的帳號其實也有資安問題。

**Comment by minniewei** [comment]
網路上充滿了危險還是小心為妙

**Comment by Lai27330** [comment]
謝謝分享，不能因為一時方便就疏忽了資安問題。
